thirdeye-spi/src/main/java/ai/startree/thirdeye/spi/detection/AnomalyFeedback.java

```diff
@@ -13,6 +13,8 @@
  */
 package ai.startree.thirdeye.spi.detection;
 
+import java.util.List;
+
 public interface AnomalyFeedback {
```

Comment:
this interface is legacy does and not have multiple implems anymore, can you remove it? This can be done as a separate PR. (can be done before or after as you want). Note I think removing it also requires changes in startree-thirdeye.

Reply:
Yeah I felt like this interface is unnecessary during the implementation. We can do it in a separate PR.